src/app/add-item/add-item.component.html

```diff
@@ -27,7 +28,12 @@ <h2 class="dialogTitle" mat-dialog-title>{{ title }}</h2>
         </mat-form-field>
     </mat-dialog-content>
 </div>
-<mat-dialog-actions class="buttons">
+<mat-dialog-actions class="buttons" *ngIf="createDialog">
     <button class="matRaisedButton close" type="button" mat-button mat-dialog-close>Close</button>
     <button class="matRaisedButton create" [disabled]="dialogForm.invalid" (click)="createActionItem()">Create<span class="faButton" *ngIf="isCreatingActionItem"><i class="fa fa-spinner fa-spin"></i></span></button>
+</mat-dialog-actions>
+
+<mat-dialog-actions class="buttons" *ngIf="!createDialog">
```

Comment:
**SUGGESTION:** This could be merged into single html and use `Save` method that underneath in TypeScript code depending on the inputs will decide if it's Create or Edit mode.